.NET/Microsoft.Recognizers.Text.DateTime/Turkish/Parsers/TurkishSetParserConfiguration.cs

```diff
@@ -80,23 +80,23 @@ public bool GetMatchedDailyTimex(string text, out string timex)
         {
             var trimmedText = text.Trim();
 
-            if (trimmedText.Equals("daily"))
+            if (trimmedText.Equals("günlük"))
```

Comment:
Please move such hardcoded values to YAML.

Reply:
done (added named groups to PeriodicRegex and DateUnitRegex and used those regexes to extract the values).